app/src/main/kotlin/com/hippo/ehviewer/ui/reader/PagerItem.kt

```diff
@@ -55,16 +54,6 @@ fun PagerItem(
 ) {
     LaunchedEffect(Unit) {
         pageLoader.request(page.index)
-        page.statusFlow.drop(1).collect {
-            if (page.statusFlow.value == PageStatus.Queued) {
-                pageLoader.request(page.index)
-            }
-        }
```

Comment:
`LaunchedEffect` will not relaunch when page loader restarts, so this is needed.

Reply:
? That does not work